server/core/src/main/java/dev/slimevr/config/TrackerConfig.kt

```diff
@@ -0,0 +1,38 @@
+package dev.slimevr.config
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.JsonNodeFactory
+import dev.slimevr.tracking.trackers.Tracker
+import io.github.axisangles.ktmath.ObjectQuaternion
+
+class TrackerConfig {
+	var customName: String? = null
+	var designation: String? = null
+
+	@get:JvmName("isHide")
```

Comment:
Does this not still affect serialization? I'm not sure about the logic with it, just make sure it still works I guess?

Reply:
the original serialization was based on setHide and isHide, so it should work